server/src/main/java/org/candlepin/pki/SubjectKeyIdentifierWriter.java

```diff
@@ -36,6 +40,14 @@
      * @throws IOException thrown if error reading cert
      * @throws NoSuchAlgorithmException thrown if JcaX509ExtensionUtils can't be created
      */
-    byte[] getSubjectKeyIdentifier(KeyPair clientKeyPair, Set<X509ExtensionWrapper> extensions)
-        throws IOException, NoSuchAlgorithmException;
+    default byte[] getSubjectKeyIdentifier(KeyPair clientKeyPair, Set<X509ExtensionWrapper> extensions)
```

Comment:
I would prefer that we keep this method definition inside DefaultSubjectKeyIdentifierWriter.  The reason being that as it stands, DefaultSubjectKeyIndentifierWriter is just an empty class.